src/components/togglePlayground.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+import { AbdsSwitch } from '@abds/react-bindings';
+
+export const TogglePlayground = () => {
+  const [darkMode, setDarkMode] = useState(false);
+
+  const toggleTheme = () => {
+    const body = document.querySelector('body');
```

Comment:
we can do this through React.Context. We pass a function down for changing the top level theme in App (which is also where I would suggest placing the theme classes).